### PR TITLE
[SYN5-163] Add a signup beta banner

### PR DESCRIPTION
**Pull request by tomsonpl**
screen shot 2017-03-22 at 13 13 42
screen shot 2017-03-22 at 13 13 37

### CODE REVIEW

File: components/BetaSignUp/BetaSignUp.js [hunk 1]
```diff
@@ -0,0 +1,117 @@
...
+      },
+      toolbarList: {
+        padding: '0 12px'
+      },
+      applyBeta: {
+        color: '#fff',
+        borderBottom: '1px solid #fff',
+        cursor: 'pointer'
+      }
+
```
Comment (Flixow) [inline components/BetaSignUp/BetaSignUp.js]: Remove it

File: components/BetaSignUp/BetaSignUp.js [hunk 2]
```diff
@@ -0,0 +1,117 @@
...
+    };
+  },
+
+  renderBetaBadge() {
+    const styles = this.getStyles();
+
+    return (
+      <span style={styles.betaBadge}>Beta</span>
```
Comment (Flixow) [inline components/BetaSignUp/BetaSignUp.js]: Just add it to render method instead of create new method

File: components/TopBar/TopBar.js [hunk 3]
```diff
@@ -4,150 +4,168 @@ import { HamburgerMenuHOC, HamburgerMenu } from '../HamburgerMenu';
 import Link from '../Link';
 import HideOnLandingPage from '../HideOnLandingPage';
 import { LoggedInContent, LoggedOutContent } from '../LoggedInContent';
+import SyncanoStatusBar from '../SyncanoStatusBar';
+
+const TopBar = (props, { hamburgerMenu, modals }) => {
+  console.log('props', props);
```
Comment (Flixow) [inline components/TopBar/TopBar.js]: ...

File: components/TopBar/TopBar.js [hunk 4]
```diff
@@ -4,150 +4,168 @@ import { HamburgerMenuHOC, HamburgerMenu } from '../HamburgerMenu';
...
-            />
-            <img
-              className="close"
-              src={require('./images/close.svg')}
-              alt="hamburger icon"
-            />
+    <div>
+      <SyncanoStatusBar
+        style={styles.noTopOnBar}
+      />
```
Comment (Flixow) [inline components/TopBar/TopBar.js]: Make it oneliner

File: pages/_template.js [hunk 5]
```diff
@@ -88,14 +94,31 @@ class Template extends Component {
...
       <div>
-        <TopBar />
-        <div className="wrapper">
+        {showBetaBanner && <BetaSignUp />}
+        <TopBar 
+          showBetaBanner={showBetaBanner}
+        />
+        <div 
+          className="wrapper"
+        >
```
Comment (Flixow) [inline pages/_template.js]: Oneliner

File: pages/_template.js [hunk 6]
```diff
@@ -88,14 +94,31 @@ class Template extends Component {
...
+    const showBetaBanner = !this.state.closeBetaBanner;
 
     return (
       <div>
-        <TopBar />
-        <div className="wrapper">
+        {showBetaBanner && <BetaSignUp />}
+        <TopBar 
+          showBetaBanner={showBetaBanner}
+        />
```
Comment (Flixow) [inline pages/_template.js]: Oneliner

File: components/CloseButton/CloseButton.js [hunk 7]
```diff
@@ -0,0 +1,33 @@
...
+      width: 48,
+      height: 48,
+      padding: 12,
+      cursor: 'pointer',
+      display: 'flex',
+      alignItems: 'center',
+      justifyContent: 'center'
+    }
+  };
+  console.log('context', context);
```
Comment (gh3r) [inline components/CloseButton/CloseButton.js]: ...

File: pages/_template.js [hunk 8]
```diff
@@ -88,14 +96,26 @@ class Template extends Component {
     return _.result(_.find(helmet.metaTags, [ 'name', 'mixpanelTitle' ]), 'content');
   };
 
+  closeBetaBanner() {
+    localStorage.setItem('closeBetaBanner', true);
+    this.setState({ closeBetaBanner: true });
+  };
+
+  onApplyBeta() {
+    console.log('Apply for beta');
```
Comment (gh3r) [inline pages/_template.js]: ??

Comment (tomsonpl) [inline pages/_template.js]: that's a place holder, I am working on the referal page - the one that opens after onApplyBeta(), on a different branch.

File: components/CloseButton/CloseButton.js (hunk 7 above)
Comment (tomsonpl) [inline components/CloseButton/CloseButton.js]: didnt see this one.

**Comment by tomsonpl** [comment]
@gh3r  just merge this if the rest is ok, and these console logs will disappear with another branch 164

**Comment by gh3r** [comment]
@tomsonpl this change will take about 30 sec to change and 6 minutes to build on CI, should we really move it to the other branch?

**Pull request merged**